include/SDL3/SDL_gpu.h

```diff
@@ -750,13 +760,15 @@ typedef struct SDL_GPUDepthStencilState
 {
     SDL_bool depthTestEnable;
     SDL_bool depthWriteEnable;
-    SDL_GPUCompareOp compareOp;
     SDL_bool stencilTestEnable;
+    Uint8 padding1;
+    SDL_GPUCompareOp compareOp;
     SDL_GPUStencilOpState backStencilState;
     SDL_GPUStencilOpState frontStencilState;
     Uint8 compareMask;
     Uint8 writeMask;
     Uint8 reference;
+    Uint8 padding2;
 } SDL_GPUDepthStencilState;
```

Comment:
A technique I've seen used in other libraries where they want to make sure a struct doesn't contain any compiler-generated alignment padding is to have a static assertion:

```
static_assert(sizeof(SDL_GPUDepthStencilState) == 2 * sizeof(SDL_GPUStencilOpState) + sizeof(SDL_GPUCompareOp) + 8);
```

Another technique that can be useful at the "OK to break API" stage is to sort struct members by size, largest first.

Reply:
> Another technique that can be useful at the "OK to break API" stage is to sort struct members by size, largest first.

After this PR lands, maybe we should do that. @icculus, thoughts?